core/src/script/CGXP/tools/autoProject.js

```diff
@@ -0,0 +1,91 @@
+/**
```

Comment:
It has to be available for the Api. So it has to be OpenLayers.

On Fri, Feb 6, 2015 at 9:09 AM, Alex Saunier <notifications@github.com>
wrote:

> In core/src/script/CGXP/tools/autoProject.js
> <https://github.com/camptocamp/cgxp/pull/909#discussion_r24227332>:
>
> > @@ -0,0 +1,91 @@
> > +/**
>
> But why name it OpenLayers.AutoProjection? Isn't it some CGXP specific
> code?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/camptocamp/cgxp/pull/909/files#r24227332>.
>



-- 

*camp**to**camp*
INNOVATIVE SOLUTIONS
BY OPEN SOURCE EXPERTS

*Gillian Milani*

Ingénieur en environnement
Stagiaire

+41 21 619 10 10 (central)
+41 21 619 10 48 (direct)
*Camptocamp SA*
Quartier de l'Innovation EPFL, PSE-A
1015 Lausanne
Suisse